zstacklib/zstacklib/storage/ceph/nbd.py

```
@@ -0,0 +1,202 @@
+        """
+        try:
+            check_nbd()
+            self.get_cmd()
+            shell.call(self.cmd)
+            if fstype is not None:
+                shell.call('mkfs -F -t %s %s' % (fstype, self.nbd_dev))
+            linux.mount(self.nbd_dev, self.mount_path)
+        except Exception as e:
+            if self.nbd_dev is not None:
+                shell.call('qemu-nbd -d %s' % self.nbd_dev)
+            raise e
```

Ruff (0.15.2)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)